content/posts/0004-writing-buildpacks.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Building a Cloud Native Buildpack Isn’t as Hard as You Think
+date: "2021-02-05"
+slug: building-a-cloud-native-buildpack-isnt-as-hard-as-you-think
+author: swigmore
+---
+
+Maybe you’ve heard some chatter about Cloud Native Buildpacks, or at least the value in containerizing your applications. Buildpacks provide a seamless mechanism for doing just that. There are a few implementations of the buildpack concept available, but the general takeaways can be summarized as follows:
+
+> “Buildpacks provide a seamless experience from source code to a running container” 
+
+> “Dockerfiles aren’t the only option anymore” 
+
+> “Everything is open source and community driven”
+
```

Comment:
I think that this is all valuable information but having it up top lessens the initial impact of the blog post and obfuscates what you are planning on talking about. Which is ultimately the composable nature of buildpacks and how easy it is to write your own.

Reply:
Regarding this comment and the one below I kind of reworked the intro. See https://github.com/paketo-buildpacks/blog/pull/8/commits/f5ab2645bad7a8962d4b59bbded22925c7dd2934